chart/templates/operator.yaml

```diff
@@ -288,40 +287,54 @@ spec:
       labels:
         app: status-controller
     spec:
+      securityContext:
+        runAsNonRoot: true
+      serviceAccountName: status-sa
+      initContainers:
+      - name: create-crd
+        image: quay.io/kubestellar/kubectl:1.29.3
+        command: ['sh', '-c']
+        args:
+        - |
+          echo -n "Waiting for managedclusteraddons CRD"
+          while [ "$(kubectl get crd clustermanagementaddons.addon.open-cluster-management.io 2> /dev/null)" == "" ] ; do
+            echo -n "."
+            sleep 1
+          done
+          echo -e "\033[0;32m\xE2\x9C\x94\033[0m"
+          echo "Creating addon-status CR:"
+          kubectl apply -f - <<EOF
+          apiVersion: addon.open-cluster-management.io/v1alpha1
+          kind: ClusterManagementAddOn
+          metadata:
+            name: addon-status
+            namespace: open-cluster-management
+          spec:
+            addOnMeta:
+              description: status addon provides full status on applied resources
```

Comment:
I do not think "applied" is accurate or informative. Is this trying to refer to `kubectl apply`? That is just one alternative way to get an object created; `kubectl create` would be another.

I think "objects" would be clearer than "resources", since "resources" is often used to mean a _kind of_ object.

It might be more informative to say something about _which_ objects. If I understand correctly, this is about getting content in the `status` section of the ManifestWork objects, conveying the status from the workload objects in the WECs.

Reply:
I did not write this
Perhaps this would be better addressed in the other PR for the CR move to crds/
@pdettori can you please let me know what you want me to do?

Reply:
@francostellari this text originates from https://github.com/kubestellar/ocm-status-addon/blob/7521a376c0eb8961e12668f4c5f9f643ee3a8769/config/manager/clustermanagementaddon.yaml#L8 - at has been there from early on. The term "applied" comes from "AppliedManifestWork". I am ok to change it to whatever could be more clear. But I agree it does not need to be in this PR.

Reply:
Yes, we can work on this later.